src/CliApplication.php

```diff
@@ -0,0 +1,581 @@
+<?php
+/**
+ * Application for Joomla Webservices.
+ *
+ * @package    Webservices
+ * @copyright  Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Webservices;
+
+use Joomla\Application\AbstractCliApplication;
+use Joomla\Application\Cli\CliOutput;
+use Joomla\Input\Cli as Input;
+use Joomla\Filter\InputFilter;
+use Joomla\Registry\Registry;
+use Joomla\Session\Session;
+
+use Joomla\Authentication\Authentication;
+
+use Joomla\DI\Container;
+use Joomla\DI\ContainerAwareTrait;
+use Joomla\DI\ContainerAwareInterface;
+
+use Joomla\Utilities\ArrayHelper;
+
+use Joomla\Webservices\Api\Soap\SoapHelper;
+use Joomla\Webservices\Webservices\Factory;
+use Joomla\Webservices\Service\ApiProvider;
+use Joomla\Webservices\Service\RendererProvider;
+
+use Negotiation\Negotiator;
+
+/**
+ * Webservices bootstrap class
+ *
+ * @package     Red
+ * @subpackage  System
+ * @since       1.0
+ */
+class CliApplication extends AbstractCliApplication implements ContainerAwareInterface
+{
+	use ContainerAwareTrait;
+
+	/**
+	 * The session object
+	 *
+	 * @var    Session
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected $session;
```

Comment:
You need to update this class to remove this session code once you've merged in the current development branch. I'm assuming this is copied from there - but it got removed in #23 ages back